packages/orchestrator-ui-components/src/types.ts

```diff
@@ -140,6 +167,35 @@ export type GraphqlQueryVariables<Type> = {
     filterBy?: GraphqlFilter<Type>[];
 };
 
+type GraphQLPageInfo = {
+    hasNextPage: boolean;
+    hasPreviousPage: boolean;
+    startCursor: number;
+    totalItems: number;
+    endCursor: number;
+};
+
+export interface ProductDefinitionsResult {
+    products: {
```

Comment:
The props "{ page,  pageInfo }" could represent a generic type, since i see the same in the 2 types below too.

Reply:
True. Fixed